Formula/linux-minidisc.rb

```diff
@@ -0,0 +1,24 @@
+class LinuxMinidisc < Formula
+  desc "Free software for accessing MiniDisc devices"
+  homepage "https://github.com/glaubitz/linux-minidisc"
+  url "https://github.com/glaubitz/linux-minidisc.git", :tag => "0.9.16"
+  version "0.9.16"
+
+  head "https://github.com/glaubitz/linux-minidisc.git"
+
+  # Dependencies copied from "build/install_dependencies.sh" in the linux-minidisc repo
+  depends_on "qt5"
```

Comment:
It's going to need `pkg-config` to find its dependencies:

```suggestion
  depends_on "pkg-config" => :build
  depends_on "qt5"
```